lib/AST/Decl.cpp

```diff
@@ -7781,6 +7781,10 @@ bool AbstractFunctionDecl::isSendable() const {
   return getAttrs().hasAttribute<SendableAttr>();
 }
 
+bool AbstractFunctionDecl::isNonisolated() const {
+  return getAttrs().hasAttribute<NonisolatedAttr>();
```

Comment:
This isn't a semantic definition of "nonisolated" (one should check `getActorIsolation` instead), so I'd prefer that we not have this function. Fortunately, it's unused, so we should remove it.

Reply:
Ah I see, cool will remove 👍

Reply:
https://github.com/apple/swift/pull/59634